integration_test/third_party_apps_data/applications/couchdb/debian_ubuntu/install

```diff
@@ -5,8 +5,18 @@ sudo apt install -y curl apt-transport-https gnupg
 
 curl https://couchdb.apache.org/repo/keys.asc | gpg --dearmor | sudo tee /usr/share/keyrings/couchdb-archive-keyring.gpg >/dev/null 2>&1
 source /etc/os-release
-echo "deb [signed-by=/usr/share/keyrings/couchdb-archive-keyring.gpg] https://apache.jfrog.io/artifactory/couchdb-deb/ ${VERSION_CODENAME} main" \
+
+if [ "${VERSION_CODENAME}" = "lunar" ]; then
+  # currently ubuntu 23.04 (lunar lobster) is not supported, install jammy (ubuntu 22.04) package instead
```

Comment:
Our customers aren't going to do that, are they? Should we just disable CouchDB support on Ubuntu 23.04 instead?